teams/libs-contributors.toml

```diff
@@ -16,7 +16,9 @@ members = [
   "sunfishcode",
   "yaahc",
 ]
-alumni = []
+alumni = [
+  "shepmaster",
```

Comment:
@shepmaster was in library-reviewers from #456 to #588. Library-reviewers got deleted instead of archived, but at least the role that shepmaster played overlaps very closely with the modern libs-contributors team. Shepmaster has [over 100 assigned library reviews spanning 3 years](https://github.com/rust-lang/rust/pulls?q=is%3Apr+assignee%3Ashepmaster+is%3Aclosed) and [standard library PRs spanning even longer](https://github.com/rust-lang/rust/pulls?q=is%3Apr+author%3Ashepmaster+is%3Aclosed). In my opinion the contribution definitely makes sense to recognize as alumni on https://www.rust-lang.org/governance/teams/library#libs-contributors.